<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reg</class>
 <widget class="QDialog" name="reg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(82, 70, 255)</string>
  </property>
  <widget class="QGroupBox" name="groupBox_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>891</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="title">
    <string>User</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>871</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_user">
         <item>
          <widget class="QLabel" name="label_user">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0)</string>
           </property>
           <property name="text">
            <string>Username:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_user">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_user_2">
         <item>
          <widget class="QLabel" name="label_pass">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0)</string>
           </property>
           <property name="text">
            <string>password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_pass">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_user_3">
         <item>
          <widget class="QLabel" name="label_first_name">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0)</string>
           </property>
           <property name="text">
            <string>first_name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_first_name">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_user_4">
         <item>
          <widget class="QLabel" name="label_first_name_2">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0)</string>
           </property>
           <property name="text">
            <string>last_name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_last_name">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_user_5">
         <item>
          <widget class="QLabel" name="label_first_name_3">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0)</string>
           </property>
           <property name="text">
            <string>Birth_day:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>22</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(246, 119, 255);</string>
           </property>
           <property name="displayFormat">
            <string>yyyy/MM/dd</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="date">
            <date>
             <year>1999</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>what are you?:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_kind">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
           <property name="modelColumn">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>profile</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
</string>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>380</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
</string>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="goooo_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>137</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="goooo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>380</y>
     <width>137</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>man_code</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
